pkg/storage/unified/resource/server.go

```diff
@@ -1549,3 +1572,28 @@ func (s *server) RebuildIndexes(ctx context.Context, req *resourcepb.RebuildInde
 
 	return s.search.RebuildIndexes(ctx, req)
 }
+
+func (s *server) checkQuota(ctx context.Context, nsr NamespacedResource) {
+	ctx, span := tracer.Start(ctx, "resource.server.checkQuota", trace.WithAttributes(
+		attribute.String("namespace", nsr.Namespace),
+		attribute.String("group", nsr.Group),
+		attribute.String("resource", nsr.Resource),
+	))
+	defer span.End()
+
+	// check quotas and log for now
+	if s.overridesService != nil {
```

Comment:
We can reduce indentation of the following code by doing `if s.overridesService == nil { return }` instead.